tests/units/anta_tests/test_vlan.py

```diff
@@ -33,4 +33,32 @@
             ],
         },
     },
+    {
+        "name": "success-singal-source-found",
+        "test": VerifyDynamicVlanSource,
+        "eos_data": [{"dynamicVlans": {"evpn": {"vlanIds": [1199]}, "mlagsync": {"vlanIds": []}}}],
+        "inputs": {"source": ["evpn", "mlagsync"]},
```

Comment:
I do not see any dynamic VLANs with the input source `mlagsync`, yet the test passes.